pkg/alerts/lcid_expiration.go

```diff
@@ -90,7 +90,7 @@ var alertsInAdvance = []time.Duration{
 // If both of these are true, we're good to fire an alert off
 func shouldSendAlertForIntake(intake models.SystemIntake, now time.Time) bool {
 	// Skip intake or if it has a status of "NO GOVERNANCE"
-	if intake.Status == models.SystemIntakeStatusNOGOVERNANCE {
```

Comment:
double check

Reply:
@ClayBenson94 can double check me, but I believe that the `NOGOVERNANCE` status in this case was closer to a generic `CLOSED` state in the v2 actions than the `NOTGOVERNANCE`, which is more of a "close because this intake is not an IT governance request".

Reply:
I can actually probably provide some good news here.

I think we can likely just nix this entire code block.

This is because of the introduction of `lcidRetiresAt` as a field that we use to ignore intakes for alerts, _combined_ with the fact that all system intakes that previously were placed in the `NOGOVERNANCE` status now have a Retired Date ([source](https://github.com/CMSgov/easi-app/wiki/Production-SQL-Updates#2024-01-18-updating-system-intakes-to-have-lcid-retire-dates)) means that we no longer need to explicitly check anything like `DecisionState` or `Step`.